frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -144,6 +148,11 @@ export class TransformTopologyData {
     return this;
   }
 
+  getImageIcon(label: string): string | boolean {
+    const icon = `icon-${label}`;
+    return logos.has(icon) && icon;
```

Comment:
Yes, this is what I did initially https://github.com/openshift/console/commit/beec0ae0ef4aa2df00934f4548b7a732fcc5ca75 but then I found a few issues like the new snapshot was being generated with `builderImage` as `test-file-stub` in all cases probably since it was getting the svg path. It didn't look right to me.  But if that is okay I can revert back to the previous changes.